src/plugins/intel_gpu/tests/functional/subgraph_tests/dynamic/matmul_weights_decompression.cpp

```diff
@@ -416,8 +416,8 @@ INSTANTIATE_TEST_SUITE_P(MatMulCompressedWeights_corner_cases_big,
 const std::vector<uint64_t> group_size = {128, 256, std::numeric_limits<int64_t>::max()};
 INSTANTIATE_TEST_SUITE_P(smoke_MatMulCompressedWeights_dyn_quan,
                          MatmulWeightsDecompression,
-                         ::testing::Combine(::testing::Values(ShapeParams{{{-1, -1, 4096}, {{1024, 1, 4096}, {1, 1, 4096}, {1024, 1, 4096}}},
-                                                                            {4096, 4096}, 128}),  // shape
+                         ::testing::Combine(::testing::Values(ShapeParams{{{-1, -1, 1024}, {{1024, 1, 1024}, {1, 1, 1024}, {1024, 1, 1024}}},
+                                                                            {1024, 1024}, 128}),  // shape
```

Comment:
Is this modification targeting a certain test? or just reducing test-tensor size?

Reply:
This is just to reduce tensor size because of test test issue.